frontend/src/utils/api/questions.js

```diff
@@ -49,10 +49,12 @@ export function getAnswers(id) {
 
 /**
  * @param {String} id the id of the question being answered
- * @param {object} body the data associated with the new answer {title, body}
+ * @param {object} body the data associated with the new answer
  */
-export function postAnswer(id, body) {
-  return post(`/question/${id}`, { body })
+export function postAnswer(id, form) {
+  const ctypeHeaders = new Headers();
```

Comment:
if you call this variable headers, you can write the line 'headers' shortly after. ie

```js
return post(`/question/${id}/answers`, {headers, body:JSON.stringify(form)})
```

Reply:
Although might need to change the way we do requests so we don't interfere with any existing headers. (Something for another day).